### DISCUSSION TITLE
Reset or set the initial data to current transform of object

### DISCUSSION THREAD

**Discussion by macdroid53** [JacquesLucke/animation_nodes#1851] [Q&A]
This may be obvious, but, I don't have much experience with nodes in general...
I have transformed and applied the transform to an object.
How do I get the Object Input to set to current values as the initial values?

**Comment by OmarEmaraDev** [comment]
Not sure I understand what you mean, can you elaborate a bit?

**Comment by macdroid53** [comment]
I think it my mistake in the way I used and input and output creating a sort of endless loop.

If I do it again I'll detail it here.
